docs/index.rst

```diff
@@ -42,6 +42,20 @@ alternatively::
 
     config.include('pyramid_storage.s3')
 
+To use GCS file storage instead of storing files locally on your server (the default assumption)::
+
+    pyramid.includes =
+        pyramid_storage.gcloud
+
+alternatively::
+
+    config.include('pyramid_storage.gcloud')
+
+We're supporting these authentication methods for GCS::
```

Comment:
```suggestion
We're supporting these authentication methods for GCS:
```

Reply:
updated this, thanks :slightly_smiling_face: